src/index.html

```diff
@@ -1,13 +1,53 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>product-card cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="product-card__imac-photo"
+      >
+      <span class="product-card__name">
+        Моноблок APPLE A1419 iMac<br> 27" Retina 5K (MNED2UA/A)
+      </span>
+      <span class="product-card__code">
+        Код товара: 195434
+      </span>
+      <div class="product-card__mark">
+        <div class="product-card__mark-star">
+          <div class="product-card__mark-star-stars star-1"></div>
```

Comment:
too complicated naming, let's simplify
also name according BEM

```suggestion
        <div class="product-card__stars stars">
          <div class="stars__star stars__star--1"></div>
```